man/sd_event_add_child.xml

```diff
@@ -164,6 +164,12 @@
     event sources are installed will not be reaped by the event loop
     implementation.</para>
 
+    <para>If the <parameter>handler</parameter> to <function>sd_event_add_child()</function> is
+    <constant>NULL</constant>, and the event source fires, this will be considered a request to exit the
+    event loop. In this case, the <parameter>userdata</parameter> parameter, cast to an integer, is used for
+    the exit code passed to
```

Comment:
is used for the exit code passed to =>is passed as the exit code to ? Ditto for other man pages.